install-dependencies.sh

```diff
@@ -61,7 +61,7 @@ then
     sudo apt-get install -y "${debian_packages[@]}"
 fi
 
-go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.3.0 sigs.k8s.io/kustomize/kustomize/v3@v3.5.1
+go get sigs.k8s.io/controller-tools/cmd/controller-gen@v0.4.1 sigs.k8s.io/kustomize/kustomize/v3@v3.5.1
```

Comment:
doesn't seem to belong in vendor commit

Reply:
done

Reply:
looks like I was a bit ambiguous there - I was pointing out to the script `install-dependencies.sh` that shouldn't be in the same commit as vendor changes. At least I am used to a rule that vendor changes go separately. so I was thinking of 2 commits:
1. go.{mod,sum} vendor/
2. install-dependencies.sh or together with some other related changes